sdk/documentintelligence/tests.yml

```diff
@@ -14,8 +14,8 @@ extends:
       # As such, the canary test runs should run on a reduced matrix.
       CloudConfig:
         Prod:
-          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
-          Location: $(LOCATION)
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+          Location: 'eastus'
```

Comment:
Why cant we use the variable anymore?

Reply:
We get a warning about the `LOCATION` in live pipeline:
```
WARNING: Attempt 1 failed: Exception calling "Invoke" with "0" argument(s): "The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: The specified location '$(LOCATION)' is invalid. A location must consist of characters, whitespace, digit, or following symbols '(,)'.
```

Reply:
I believe this comes from the variables you can configure in the pipeline it's odd it's not setting correctly. I would ask eng sys about this.